endpoint/endpointComp/resolv.conf

```diff
@@ -0,0 +1 @@
+nameserver 8.8.8.8
```

Comment:
Add comment for further clean up. This file should be generated.

Reply:
Did you mean adding the comment to describe that the user can use their own `resolv.conf`?

Reply:
At least, the DNS configuration should be assigned via build system rather than hard-coded file.

Reply:
The DNS configuration is assigned via **Make** build system now.